pywinauto/unittests/test_Calendar.py

```diff
@@ -86,11 +88,102 @@ def test_can_set_calendars_view_into_century(self):
+        expected_rect = self.get_expected_minimized_rectangle()
+        rect = self.calendar.minimize_calendar_rectangle(-1, -1, -1, -1)
+        self.assertEquals(expected_rect, rect)
+
+    def test_can_determine_calendar_is_hit(self):
+        res = self.calendar.do_hit_test(4, 48)
+        self.assertEquals(win32defines.MCHT_CALENDAR, res)
+
+    def test_can_determine_calendar_background_is_hit(self):
+        res = self.calendar.do_hit_test(4, 48)
+        self.assertEquals(win32defines.MCHT_CALENDARBK, res)
+
+    def test_can_determine_date_is_hit(self):
+        res = self.calendar.do_hit_test(140, 120)
+        self.assertEquals(win32defines.MCHT_CALENDARDATE, res)
+
+    def test_can_determine_next_month_date_is_hit(self):
+        res = self.calendar.do_hit_test(140, 130)
+        self.assertEquals(win32defines.MCHT_CALENDARDATENEXT, res)
+
+    def test_can_determine_prev_month_date_is_hit(self):
+        res = self.calendar.do_hit_test(10, 60)
+        self.assertEquals(win32defines.MCHT_CALENDARDATEPREV, res)
+
+    def test_can_determine_nothing_is_hit(self):
+        res = self.calendar.do_hit_test(0, 0)
+        self.assertEquals(win32defines.MCHT_NOWHERE, res)
+
+    def test_can_determine_top_left_title_corner_is_hit(self):
+        res = self.calendar.do_hit_test(10, 10)
+        self.assertEquals(win32defines.MCHT_TITLEBTNPREV, res)
+
+    def test_can_determine_title_is_hit(self):
+        res = self.calendar.do_hit_test(20, 15)
+        self.assertEquals(win32defines.MCHT_TITLE, res)
+
+    def test_can_determine_title_background_is_hit(self):
+        res = self.calendar.do_hit_test(20, 15)
+        self.assertEquals(win32defines.MCHT_TITLEBK, res)
+
+    def test_can_determine_top_right_title_corner_is_hit(self):
+        res = self.calendar.do_hit_test(150, 20)
+        self.assertEquals(win32defines.MCHT_TITLEBTNNEXT, res)
+
+    def test_can_determine_today_link_is_hit(self):
+        res = self.calendar.do_hit_test(130, 140)
+        self.assertEquals(win32defines.MCHT_TODAYLINK, res)
+
+    def test_can_determine_day_abbreviation_is_hit(self):
+        res = self.calendar.do_hit_test(30, 40)
+        self.assertEquals(win32defines.MCHT_CALENDARDAY, res)
+
+    def test_can_determine_week_number_is_hit(self):
+        self.set_calendar_state_to_display_week_numbers()
+        res = self.calendar.do_hit_test(4, 50)
+        self.assertEquals(win32defines.MCHT_CALENDARWEEKNUM, res)
+
     def assert_system_time_is_equal_to_current_date_time(self,systemTime, now):
         self.assertEqual(systemTime.wYear, now.year)
         self.assertEqual(systemTime.wMonth, now.month)
         self.assertEqual(systemTime.wDay, now.day)
 
+    def get_expected_minimized_rectangle(self):
+        expected_rect = win32structures.RECT()
+        expected_rect.left = 0
+        expected_rect.top = 0
+        expected_rect.right = 162
+        expected_rect.bottom = 160
```

Comment:
Using hard coded coordinates here and in previous tests with ```do_hit_test``` isn't reliable. The control size and coordinates change under different windows themes. We already have this problem in other tests. Maybe you should try to get the main control dimensions before testing and use them as the basis for the rest?